features/realtime_kernel.feature

```diff
@@ -0,0 +1,146 @@
+            | release | version    | full_name     |
+            | xenial  | 16.04 LTS  | Xenial Xerus  |
+            | bionic  | 18.04 LTS  | Bionic Beaver |
+            | focal   | 20.04 LTS  | Focal Fossa   |
+
+    @series.jammy
+    @uses.config.machine_type.gcp.generic
+    Scenario Outline: Enable Real-Time Kernel service
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        When I attach `contract_token_staging` with sudo
+        Then I verify that running `ua enable realtime-kernel` `as non-root` exits `1`
+        And I will see the following on stderr:
+            """
+            This command must be run as root (try using sudo).
+            """
+        Then I verify that running `ua enable realtime-kernel` `with sudo` exits `1`
+        And stderr matches regexp:
+            """
+            Cannot enable unknown service 'realtime-kernel'.
+            """
+        When I run `ua enable realtime-kernel --beta` `with sudo` and stdin `y`
+        Then stdout matches regexp:
+            """
+            One moment, checking your subscription first
+            The real-time kernel is a beta version of the 22.04 Ubuntu kernel with the
+            PREEMPT_RT patchset integrated for x86_64 and ARM64. If you haven’t already,
+            please register for this free beta program at https://ubuntu.com/realtime-kernel.
+
+            .*You will not be able to revert to your original kernel after enabling real-time..*
+
+            Do you want to continue\? \[ default = Yes \]: \(Y/n\) Updating package lists
+            Installing Real-Time Kernel packages
+            Real-Time Kernel enabled
+            A reboot is required to complete install.
+            """
+        When I run `apt-cache policy linux-realtime` as non-root
+        Then stdout does not match regexp:
+            """
+            .*Installed: \(none\)
+            """
+        And stdout matches regexp:
+            """
+            \s* 500 https://esm.staging.ubuntu.com/realtime/ubuntu <release>/main amd64 Packages
+            """
+        When I verify that running `ua enable realtime-kernel --beta` `with sudo` exits `1`
+        Then stdout matches regexp
+            """
+            One moment, checking your subscription first
+            Real-Time Kernel is already enabled.
+            See: sudo ua status
+            """
+        When I reboot the `<release>` machine
+        When I run `uname -r` as non-root
+        Then stdout matches regexp:
+            """
+            realtime
+            """
+        When I run `ua disable realtime-kernel` `with sudo` and stdin `y`
+        Then stdout matches regexp:
+            """
+            This will disable the Real-Time Kernel entitlement but the Real-Time Kernel will remain installed.
+            """
+        Examples: ubuntu release
+            | release |
+            | jammy   |
+
+# TODO uncomment and test when jammy supports livepatch
```

Comment:
I think we can remove this bit here and just have a follow up PR for it

Reply:
As soon as this is merged and deployed https://github.com/canonical/ua-contracts/pull/2731 then we should include this test. It is so close to happening that I'd rather leave this here. It may even happen before we land this PR anyway

Reply:
@orndorffgrant I think we can add this test back

Reply:
@lucasmoura ended up just removing it since livepatch on jammy isn't ready yet